src/installing-deis/configuring-object-storage.md

```diff
@@ -15,10 +16,42 @@ Additionally, Deis ships with a [Minio](http://minio.io) [component](https://git
 
 # Telling Deis What to Use
 
-The Deis components determine what object storage system to use via environment variables that you set up:
+The Deis components determine what object storage system to use via environment variables that you set up. The below list is the lookup order for all Deis components.
```

Comment:
@arschles Im good with the PR if you can some how add a special column for builder storage settings. This heading says the below settings are for deis which is not true.